styles/sas-trading.css

```diff
@@ -36,6 +36,35 @@
     gap: 1em;
 }
 
-.sas-menu-overview-table-data {
+.sas-menu-mini-table {
+    width: 90%;
+    margin-left: 6%;
+}
+
+.sas-menu-table-data {
     text-align: center;
 }
+
+.sas-menu-table-left-column {
```

Comment:
Might want to add "mini" to these to indicate they're usually used with `sas-menu-mini-table`

Reply:
Changed